### ISSUE TITLE
TERRA-45 ⁃ astra_access_list only creates the first address

### ISSUE THREAD

**Issue by edmeister** [datastax/terraform-provider-astra#53]
### Terraform Version
Terraform core: `1.0.11`
Astra provider: `1.0.12`

### Affected Resource(s)
Please list the resources as a list, for example:

* astra_access_list

### Terraform Configuration Files
```hcl
module "astra" {
  ...
  private_link_access_cidr = [
    "10.0.0.1/32",
    "10.0.0.2/32",
    "10.0.0.3/32",
  ]
}

# actual module
variable "private_link_access_cidr" {
  type        = list(string)
  description = "List of CIDR's with access to the private link"
  default     = []
}

resource "astra_database" "database" {
  ...
}

resource "astra_access_list" "access_list" {
  database_id = astra_database.database.id

  addresses {
    dynamic "request" {
      for_each = var.private_link_access_cidr
      content {
        address = request.value
        enabled = true
      }
    }
  }
}

```

### Expected Behavior
The database should have an access list with 3 records.

### Actual Behavior
The terraform plan shows the resource as it is intended:

```
# module.astra.astra_access_list.access_list will be created
  + resource "astra_access_list" "access_list" {
      + database_id = (known after apply)
      + id          = (known after apply)

      + addresses {
          + request {
              + address = "10.0.0.1/32"
              + enabled = true
            }
          + request {
              + address = "10.0.0.2/32"
              + enabled = true
            }
          + request {
              + address = "10.0.0.3/32"
              + enabled = true
            }
        }
    }
```

Only the first address is created. See screenshot below.

![image](https://user-images.githubusercontent.com/702527/143002580-62bf715c-9756-40c6-8b22-ad7367f59a27.png)

The tfstate contains the 3 records:

```
    {
      "module": "module.astra",
      "mode": "managed",
      "type": "astra_access_list",
      "name": "access_list",
      "provider": "provider[\"registry.terraform.io/datastax/astra\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "addresses": [
              {
                "request": [
                  {
                    "address": "10.0.0.1/32",
                    "description": "",
                    "enabled": true
                  },
                  {
                    "address": "10.0.0.2/32",
                    "description": "",
                    "enabled": true
                  },
                  {
                    "address": "10.0.0.3/32",
                    "description": "",
                    "enabled": true
                  }
                ]
              }
            ],
            "database_id": "*****************",
            "id": "*******************"
          },
          "sensitive_attributes": [],
          "private": "bnVsbA==",
          "dependencies": [
            "module.astra.astra_database.database"
          ]
        }
      ]
    },
```

### Steps to Reproduce
Just apply the code on a new database.

┆Issue is synchronized with this [Jira Task](https://datastax-oss.atlassian.net/browse/TERRA-45) by [Unito](https://www.unito.io)
┆friendlyId: TERRA-45
┆priority: Major

**Comment by emerkle826** [comment]
Thanks for reporting this. I see the issue and should be able to resolve this shortly

**Comment by emerkle826** [comment]
Should be resolved in v1.0.13, which will be available shortly.

**Comment by edmeister** [comment]
Works! Thanks, @emerkle826
